core/zoom_controls.js

```diff
@@ -207,16 +206,11 @@ Blockly.ZoomControls.prototype.createZoomOutSvg_ = function(rnd) {
       },
       this.zoomOutGroup_);
   zoomoutSvg.setAttributeNS(Blockly.utils.dom.XLINK_NS, 'xlink:href',
-      ws.options.pathToMedia + Blockly.SPRITE.url);
+      this.workspace_.options.pathToMedia + Blockly.SPRITE.url);
 
   // Attach listener.
-  Blockly.bindEventWithChecks_(zoomoutSvg, 'mousedown', null, function(e) {
-    ws.markFocused();
-    ws.zoomCenter(-1);
-    Blockly.Touch.clearTouchIdentifier();  // Don't block future drags.
-    e.stopPropagation();  // Don't start a workspace scroll.
-    e.preventDefault();  // Stop double-clicking from selecting text.
-  });
+  Blockly.bindEventWithChecks_(
```

Comment:
Not necessarily in this PR, but ideally we should be maintaining a reference to this and disposing of the event handler in the zoom control `dispose` method.

Reply:
sgtm I'll make a follow up PR to address this and add you to it